web/survey/models.py

```diff
@@ -624,6 +551,42 @@ class Meta:
         db_table = 'tile_obs'
 
 
+class SpatialRefSys(models.Model):
```

Comment:
Do these new tables apply only to WALLABY?

Reply:
I don't know the point of the `SpatialRefSys` table. I think this was just always in the codebase since I've been around and I haven't removed it. The survey component tables could be used for other surveys (to allow repeated detections for different mosaics for example in DINGO).

The source extraction regions table could be WALLABY specific. Do you use these in DINGO?

Reply:
It isnt used in DINGO

Reply:
I've removed `SpatialRefSys` and have moved other tables to be WALLABY specific

Reply:
Those other tables can't be placed under a WALLABY check if statement since they're imported throughout the code. I've left them in the main section of `models.py`